secrets.env.template

```diff
@@ -0,0 +1,15 @@
+# Document Intelligence
+AZURE_API_KEY=
+
+# OpenAI
+AZURE_OPENAI_KEY=
+
+# API base path for deployment
+API_BASE_PATH='/swagger'
+SWAGGER_PATH='/docs'
+
+# DB
+FERTISCAN_DB_URL=
+
+# Blob Storage
+FERTISCAN_STORAGE_URL=
```

Comment:
Do you think it would be possible to store both the username and password instead of the full URL for the `secrets.env.template` file? Full url's are quite hard to manage with the `git-secrets` tool

Reply:
I agree we should be splitting this up and building the string in code at runtime if we need to use the string